gettsim/shared.py

```diff
@@ -59,24 +122,23 @@ def get_names_of_arguments_without_defaults(function):
     return argument_names_without_defaults
 
 
-def add_rounding_spec(params_key):
-    """Decorator adding the location of the rounding specification to a function.
+def remove_group_suffix(col):
 
-    Parameters
-    ----------
-    params_key : str
-        Key of the parameters dictionary where rouding specifications are found. For
-        functions that are not user-written this is just the name of the respective
-        .yaml file.
+    # Set default result
+    out = col
 
-    Returns
-    -------
-    func : function
-        Function with __rounding_params_key__ attribute
-    """
+    # Remove suffix from result if applicable
+    for g in SUPPORTED_GROUPINGS:
+        if col.endswith(f"_{g}"):
+            out = rchop(col, f"_{g}")
 
-    def inner(func):
-        func.__rounding_params_key__ = params_key
-        return func
+    return out
 
-    return inner
+
+def rchop(s, suffix):
+    # ToDO: Replace by removesuffix when only python >= 3.9 is supported
```

Comment:
I think we can safely assume this for the next release. Can you add an issue to remove support for 3.8 and add support for 3.11? Leave a note there to delete this function.